src/components/forms/PractitionerForm/UserIdSelect/index.tsx

```diff
@@ -81,27 +87,28 @@ export const UserIdSelect = (props: Props) => {
   };
 
   /** Pulls all users data */
-  const loadUsers = async (service: typeof OpenSRPService = OpenSRPService) => {
+  const loadUsers = async () => {
     let filterParams = {
       page_size: USERS_REQUEST_PAGE_SIZE,
```

Comment:
also please check that the deault `USERS_REQUEST_PAGE_SIZE` is set to a `1000`

Reply:
addressed via https://github.com/onaio/reveal-frontend/pull/1166/commits/5021264b457440efccc9f7e22ce84498b4b5e093